basedecorator.rb

```diff
@@ -0,0 +1,12 @@
+require 'nameable'
+
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super

```

Comment:
* Kindly add `super( )` as I get an argument error as `nameable is not expecting any argument [Kindly read more on the super keyword](https://www.rubyguides.com/2018/09/ruby-super-keyword/) 
![image](https://user-images.githubusercontent.com/52497246/213361057-e4885f75-cc02-4723-bac9-2f002283aa33.png)

Reply:
* This is what i get after fixing the issue 
![image](https://user-images.githubusercontent.com/52497246/213361731-f64e3a72-a03e-4e87-8858-4ab6245d1479.png)